nixos/modules/hardware/video/nvidia.nix

```diff
@@ -249,7 +249,7 @@ in
       modules = optional (igpuDriver == "amdgpu") [ pkgs.xorg.xf86videoamdgpu ];
       deviceSection = ''
         BusID "${igpuBusId}"
-        ${optionalString syncCfg.enable ''Option "AccelMethod" "none"''}
+        ${optionalString (syncCfg.enable && igpuDriver != "amdgpu") ''Option "AccelMethod" "none"''}
```

Comment:
Think this may be unneeded for all setups, the nvidia docs don't recommend it.

https://download.nvidia.com/XFree86/Linux-x86_64/510.39.01/README/randr14.html - current
https://download.nvidia.com/XFree86/Linux-x86_64/361.18/README/randr14.html - very old

@eadwu do you remember why it was included originally?

Reply:
I do not. This is was something that was carried over from the initial code. It *might've* been on the Arch wiki since I do recall seeing something like this somewhere once upon a time.

https://github.com/NixOS/nixpkgs/pull/42846/files

Reply:
Cool, can probably remove this in a followup change.

I don't want to ask @GoogleBot42 to include that in this change as a few people have already tested it and it'd be good to get it merged.

Reply:
yeah i think it can be removed.  My understanding is it is not needed. But since I don't have an intel+nvidia machine to test with I wasn't comfortable making the change.